src/components/Swap/Summary/index.test.tsx

```diff
@@ -47,6 +47,8 @@ function getInitialTradeState(trade: Partial<Swap> = {}) {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const EMPTY_FUCTION = () => {}
```

Comment:
you could use `jest.fn()` here too

Reply:
also nit: these empty functions are typically just named `noop` xD